packages/fxa-graphql-api/src/gql/lib/error.ts

```diff
@@ -4,20 +4,7 @@
 
 import { ApolloError } from 'apollo-server';
 
-export class ThrottledError extends ApolloError {
-  constructor(
-    message: string,
-    code: string,
-    extensions: {
-      errno: number;
-      info: string;
-      retryAfter: number;
-      retryAfterLocalized: string;
-    }
-  ) {
-    super(message, code, extensions);
-  }
-}
+export class ThrottledError extends ApolloError {}
```

Comment:
I don't have enough context to say whether this is temporary, necessary, or whatever.  So just a note in case it needs to be reverted or something.

Reply:
The constructor tripped a 'useless constructor' warning, since it's identical to the base classes constructor.